README.md

```diff
@@ -13,18 +13,12 @@
 
 ## 🧑‍🏫 Rules
 
-- [x] use datetime instead of days_ago in DAG start_date
-- [x] task decorator instead of PythonOperator and PythonVenvOperator
-- [x] dag decorator instead of DAG
-- [x] jinja string instead of Variable.get
+- [x] Use function-level imports instead of top-level imports[^1] (see [Top level Python Code](https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#top-level-python-code))
```

Comment:
@isidentical In Airflow, not sure if you know the tool, you should avoid top-level code especially imports can cause huge performance issues as Airflow has a parser that continously parses the python file.